forms/collections/elements.js

```diff
@@ -12,6 +12,7 @@ define(function (require) {
   var formsErrors = require('forms/error-helpers');
 
   // this module
+  var ElementsCollection;
```

Comment:
there was a need to refer to the collection as i wanted the error object to be an element collection as well, however because we dont uniquely identify elements inside a subform (they all have the same name) a backbone collection couldnt be used. The var declaration is an artifact